packages/gitmoji-changelog-cli/src/cli.js

```diff
@@ -131,4 +137,62 @@ function logMetaData(changelog) {
   }
 }
 
+async function handleUnexpectedErrors(options, projectInfo, repository, e) {
+  const envInfo = await envinfo.run(
+    {
+      System: ['OS'],
+      Binaries: ['Node', 'Yarn', 'npm'],
+    },
+    { markdown: true }
+  )
```

Comment:
Later when we will have more preset, `Manager > Maven`, `Manager > Gradle` could be interesting